cloudmarker/events/azsqldatabasedisabledtdeevent.py

```diff
@@ -0,0 +1,95 @@
+"""Microsoft Azure SQL DB Transparent Data Encryption (TDE) event.
+
+This module defines the :class:`AzSQLDatabaseDisabledTDEEvent` class that
+identifies if a SQL Database has TDE disabled . This plugin works on the
+SQL DB properties found in the ``ext`` bucket of ``sql_db`` records.
+"""
+
+
+import logging
+
+from cloudmarker import util
+
+_log = logging.getLogger(__name__)
+
+
+class AzSQLDatabaseDisabledTDEEvent:
+    """Az SQL Database TDE event plugin."""
+
+    def __init__(self):
+        """Create an instance of :class:`AzSQLDatabaseDisabledTDEEvent`."""
+
+    def eval(self, record):
+        """Evaluate Azure SQL DB for disabled TDE.
+
+        Arguments:
+            record (dict): A SQL DB record.
+
+        Yields:
+            dict: An event record representing an Azure SQL DB with
+            TDE disabled
```

Comment:
Please end this with full stop for consistency.

Reply:
Fixed